### PR TITLE
Add translation support

### PR DESCRIPTION
**Pull request by InFTord**
Adds support for translating strings. Example: Command descriptions can be translated in different languages with that change now.
Right now our setup works via Crowdin (https://crowdin.com/project/fabrissentials/ - link, but it's probably temporary)
Still, we need to discuss how we will format everything.

### CODE REVIEW

**Comment by InFTord** [comment]
i had too much skill issue meanwhile doing this PR :skull:

**Comment by InFTord** [comment]
Speaking of formatting, how are we gonna format stuff like:
`{
    "command.test.testing": "Test"
}
`

**Comment by OoLunar** [comment]
> Speaking of formatting, how are we gonna format stuff like:
> 
> ```json
> {
>     "command.test.testing": "Test"
> }
> ```

Are these the keys used to get the translations? If so, yeah that's fine.
---
I can see that we've added CrowdIn as a dependency. Is this a compile time dependency or runtime? Do I need to add any Github Secrets to make CrowdIn work? I believe I need to create a project on their website, correct?

**Comment by InFTord** [comment]
Yea, you need to create a project, but i need to delete my own project right now.

File: gradle.properties [hunk 1]
```diff
@@ -14,4 +14,5 @@ maven_group=net.forsaken_borders
 archives_base_name=fabrissentials
 
 # Dependencies
-fabric_version=0.73.0+1.19.3
\ No newline at end of file
+fabric_version=0.73.0+1.19.3
+3
```
Comment (Aaron2550) [inline gradle.properties:18]: 3

**Review by Aaron2550 — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (InFTord) [reply gradle.properties:18]: 3 indeed :wheeze:

**Review by OoLunar — APPROVED** [review]
LGTM, just make sure that nothing was lost during the rebase.

**Pull request merged**